packages/web-forms/src/components/FormLanguageDialog.vue

```diff
@@ -0,0 +1,102 @@
+<script setup lang="ts">
+import type { FormLanguage, RootNode } from '@getodk/xforms-engine';
+import PrimeButton from 'primevue/button';
+import PrimeDialog from 'primevue/dialog';
+import PrimeRadioButton from 'primevue/radiobutton';
+import { ref } from 'vue';
+
+const props = defineProps<{ form: RootNode, state: boolean }>();
+const emit = defineEmits(['update:state']);
+
+const languages = props.form.languages.filter(language => !language.isSyntheticDefault);
+
+const selectedLanguage = ref<FormLanguage>();
+
+const initSelectedLanguage = () => {
+	if(!props.form.currentState.activeLanguage.isSyntheticDefault) {
+		selectedLanguage.value = props.form.currentState.activeLanguage;
+	}
+}
+
+initSelectedLanguage();
```

Comment:
maybe do a check in parent component about isSyntheticDefault

Reply:
One thing that I didn't anticipate when we were chatting about this yesterday, but also worth mentioning: when a form is **not translated**, the mobile presentation currently moves the print button into a menu that has no other items. That print button takes up the same space as the hamburger, so it's not really saving any space.

I don't think we necessarily need to change anything on this now, but it's one more thing to add to the discussion of what to do with the print button in mobile. And what we do if/when there are more header controls like view toggle.